### DISCUSSION TITLE
/posts/2024/01/26Nuxt3%E6%89%8B%E5%86%99%E4%B8%80%E4%B8%AA%E6%90%9C%E7%B4%A2%E9%A1%B5%E9%9D%A2

### DISCUSSION THREAD

**Discussion by giscus** [Justin3go/justin3go.com#77] [Announcements]
# /posts/2024/01/26Nuxt3%E6%89%8B%E5%86%99%E4%B8%80%E4%B8%AA%E6%90%9C%E7%B4%A2%E9%A1%B5%E9%9D%A2

坚持深耕技术领域的T型前端程序员, 关注独立开发与AI，喜欢Vuejs、Nestjs, 还会点Python、搜索引擎、NLP、Web3、后端

https://justin3go.com/posts/2024/01/26Nuxt3%E6%89%8B%E5%86%99%E4%B8%80%E4%B8%AA%E6%90%9C%E7%B4%A2%E9%A1%B5%E9%9D%A2

**Comment by Zding89** [comment]
挺喜欢简洁界面，但单纯的采集搜索站已经很多了。要是能做个后台，关联自己的token，别人提取能先转存到自己空间，再分享出新地址，就能推给身边朋友用，刷刷达人了

**Comment by Justin3go** [comment]
这个评论有点问题，不知道为什么会覆盖主页的评论，我先更改讨论类型，后面再仔细排查下为什么

**Reply by Justin3go** [reply]
现在应该解决了

**Comment by fankter** [comment]
为啥我用手机浏览器访问不了，打开就500，E3.at is not a function

**Reply by Justin3go** [reply]
能否麻烦截个图看看，以及用得啥浏览器

**Reply by fankter** [reply]
![20240407174410](https://github.com/Justin3go/justin3go.github.io/assets/141980026/edf765a8-809f-4217-a750-f35e1dc046cc)
小米手机自带的浏览器

**Reply by Justin3go** [reply]
可能是这个浏览器不兼容一些新的方法，我也没有做降级兼容处理（暂无打算去优化这个），可以试试其他比较主流的浏览器，如Chrome、Edge、Firefox之类的
